cmd/dendritejs/main.go

```diff
@@ -197,14 +196,12 @@ func main() {
 	deviceDB := base.CreateDeviceDB()
 	federation := createFederationClient(cfg, node)
 
-	serverKeyAPI := serverkeyapi.SetupServerKeyAPIComponent(
-		base, federation,
-	)
+	fetcher := &libp2pKeyFetcher{}
 	keyRing := gomatrixserverlib.KeyRing{
 		KeyFetchers: []gomatrixserverlib.KeyFetcher{
-			&libp2pKeyFetcher{},
+			fetcher,
```

Comment:
I think if you already have `fetcher` satisfying `KeyDatabase` then you probably don't need to specify any `KeyFetchers` at all.

Reply:
Can't hurt to have it?

Reply:
Won't do any harm, nope. It'll probably never get hit.

Reply:
That's what she said